data/DiaSource.yaml

```diff
@@ -86,8 +93,42 @@ funcs:
     # trailLnL not implemented
     # trailChi2 not implemented
     # trailNdata not implemented
-    # dipMeanFlux needs functor DM-
-    # dipFluxDiff needs functor DM-
+    dipMeanFlux:
```

Comment:
Also, could you please put a note at the top of the file referencing or linking to where these fields are defined? They don't have docstrings in here, and I assume they are formally defined elsewhere. Not knowing the package, I don't know where to look that up.

Reply:
Added reference to http://ls.st/dpdd